emsdk.py

```diff
@@ -1663,8 +1663,14 @@ def generate_dot_emscripten(active_tools):
       print('    ' + p)
     print('- This can be done for the current shell by running:')
     print('    source "%s"' % emsdk_env)
-    print('- Configure emsdk in your bash profile by running:')
-    print('    echo \'source "%s"\' >> $HOME/.bash_profile' % emsdk_env)
+    print('- Configure emsdk in your shell startup scripts by running:')
+    shell = os.environ.get('SHELL', '')
+    if 'zsh' in shell:
```

Comment:
should these perhaps be `shell.endswith('/zsh')`? but I guess it's almost always `/bin/X` anyhow...